Source/main.cpp

```diff
@@ -161,6 +162,9 @@ main (int   argc,
 
     }
 
+    Castro* castro = dynamic_cast<Castro*>(&amrptr->getAmrLevels()[0]);
```

Comment:
Because post_simulation is a static function, it can be called directly,

Castro::post_simulation(amrptr->getAmrLevels());